helm-charts/splunk-otel-collector/templates/network-explorer/kernel-collector-podsecuritypolicy.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if and .Values.networkExplorer.enabled .Values.networkExplorer.podSecurityPolicy.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
PSPs are removed from kubernetes in 1.25, see https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future. Can we do not include it?